components/browser/errorpages/src/main/res/raw/error_pages.html

```diff
@@ -26,7 +26,7 @@ <h1 id="et_dnsNotFound" class="errorTitleText">%messageShort%</h1>
 
                 <!-- Short Description -->
                 <div id="errorShortDesc">
-                    <p id="errorShortDescText">%messageLong%</p>
```

Comment:
Do you know why we were using this id `errorShortDescText`? Not sure if removing it could cause any breakage to some JavaScript or CSS that relies on it.

Reply:
There is no reference in CSS or code to this. The styling is done in the `<div>`. 
Either way, if it was used, it wouldn't work as expected. When parsing the code, after the `<ul>` element a `</p`> would be automatically inserted by the parser.

Reply:
👍